import-export-cli/integration/envSpecific_test.go

```diff
@@ -149,9 +148,9 @@ func TestEnvironmentSpecificParamsEndpointSecurityFalse(t *testing.T) {
 	testutils.ValidateAPIsEqual(t, api, importedAPI)
 }
 
-func TestEnvironmentSpecificParamsEndpointSecurityDigest(t *testing.T) {
-	superTenantAdminUsername := superAdminUser
-	superTenantAdminPassword := superAdminPassword
+func TestEnvironmentSpecificParamsEndpointSecurityDigestDevopsSuperTenant(t *testing.T) {
```

Comment:
Shall we add a comment explaining the test case a bit as in other test cases[1]?

[1]. https://github.com/wso2/product-apim-tooling/blob/47ed2a862e5fc676c9f8ce801e885d9229fa0763/import-export-cli/integration/app_test.go#L484

Reply:
Fixed via e4f37dba86efef403f4a1d3967821d1f2791019b